.github/workflows/run_pytest.yaml

```diff
@@ -0,0 +1,27 @@
+name: PyTest
+
+on:
+  # code review question: this commits directly to PR branches.
+  # that seems like what we're looking for, but we could also format only
+  # committed code.
```

Comment:
I don't understand that. The action is just running test. Why are are talking about "commiting" and "formatting"?

Reply:
Copied it from somewhere else. Not applicable here.